test/compflags/bradc/printstuff/versionhelp.sh

```diff
@@ -4,7 +4,8 @@ compiler=$3
 
 echo -n `basename $compiler`
 cat $CWD/version.goodstart
-echo ""
+diff $CWD/../../../../compiler/main/BUILD_VERSION $CWD/zero.txt > /dev/null 2>&1 && echo "" || \
+    { echo -n " pre-release (" && cat $CWD/../../../../compiler/main/BUILD_VERSION | tr -d \"\\n && echo ")" ; }
```

Comment:
Could simplify the `cat ... | tr` pipe to just `tr -d \"\\n < $CWD/../../../../compiler/main/BUILD_VERSION`.

Reply:
I didn't try to be at all creative here, but just restored the lines that were here for the 1.23 pre-release (or at least, that's what I think I did).